autonat.go

```diff
@@ -90,6 +103,18 @@ func (as *AmbientAutoNAT) Status() NATStatus {
 	return as.status
 }
 
+func (as *AmbientAutoNAT) updateStatus(s NATStatus) {
+	as.status = s
+	switch s {
+	case NATStatusUnknown:
+		as.emitUnknown.Emit(event.EvtLocalRoutabilityUnknown{})
```

Comment:
We really need to make channels buffered by default in the eventbus, because otherwise these emits could block.

Reply:
Fixed by https://github.com/libp2p/go-eventbus/pull/13